### PR TITLE
Upgrade to MR openxr plugin to version 1.4.0

### PR DESCRIPTION
**Pull request by yl-msft**

### CODE REVIEW

**Review by keveleigh — APPROVED** [review]

File: BasicSample/ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.28f1
-m_EditorVersionWithRevision: 2020.3.28f1 (f5400f52e03f)
+m_EditorVersion: 2020.3.31f1
+m_EditorVersionWithRevision: 2020.3.31f1 (6b54b7616050)
```
Comment (satyapoojasama) [inline BasicSample/ProjectSettings/ProjectVersion.txt:2]: is this the recommended version of unity for this project?

File: BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs [hunk 2]
```diff
@@ -48,7 +48,7 @@ async void Start()
             }
             m_arAnchorManager.anchorsChanged += AnchorsChanged;
 
-            m_anchorStore = await XRAnchorStore.LoadAsync(m_arAnchorManager.subsystem);
+            m_anchorStore = await m_arAnchorManager.LoadAnchorStoreAsync();
```
Comment (satyapoojasama) [inline BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs:51]: also just curious how did pipeline have a successful basic sample build in CI without this change?

File: BasicSample/ProjectSettings/ProjectVersion.txt (hunk 1 above)
Comment (yl-msft) [reply BasicSample/ProjectSettings/ProjectVersion.txt:2]: it's the latest that works for HL2.

File: BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs (hunk 2 above)
Comment (yl-msft) [reply BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs:51]: This is a warning instead of error.

**Review by KBonds — APPROVED** [review]

**Review by satyapoojasama — APPROVED** [review]

**Pull request merged**